content/docs/user-guide/assets/pipeline/asset-cache.md

```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
+
 ### Asset Cache
 
- The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+
+The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+
+In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
 
- The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+Asset Processor assumes it is the only process that makes changes to the Asset Cache. If you modify or delete files in the Asset Cache, including deleting product asset files while Asset Processor is not running. the changes might not be noticed by Asset Processor.
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. The default behavior for release builds is to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)

```

Comment:
nit: The Asset Cache is intended to only be used only during development. By default the Editor and game launchers in non-release builds load assets from the asset cache.

There is not a notion of "development" build, so be a little careful with this term. Users have debug, profile or release builds, which they can use in their development "phase", we also have a "development branch" for O3DE.